src/RazorSdk/Tasks/Microsoft.NET.Sdk.Razor.Tasks.csproj

```diff
@@ -71,6 +71,9 @@
 
   <ItemGroup>
     <Compile Include="**\*.cs" />
+    <Compile Include="..\Tool\CommandLine\*.cs">
```

Comment:
Do you need all the files from this directory?

Reply:
Nope. Just the `ArgumentEscaper`. I can be more specific here.